<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LightWidgetForm</class>
 <widget class="QWidget" name="LightWidgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>453</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(189, 189, 189);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Light</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Ambient</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="AmbientColorButton">
             <property name="text">
              <string>Color</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../../Users/KOSA/Desktop/image/LightColor.png</normaloff>../../Users/KOSA/Desktop/image/LightColor.png</iconset>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextUnderIcon</enum>
             </property>
             <property name="arrowType">
              <enum>Qt::NoArrow</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QSlider" name="AmbientSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Diffuse</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="DiffuseColorButton">
             <property name="text">
              <string>Color</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../../Users/KOSA/Desktop/image/LightColor.png</normaloff>../../Users/KOSA/Desktop/image/LightColor.png</iconset>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextUnderIcon</enum>
             </property>
             <property name="arrowType">
              <enum>Qt::NoArrow</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QSlider" name="DiffuseSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Specular</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="SpecularColorButton">
             <property name="text">
              <string>Color</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../../Users/KOSA/Desktop/image/LightColor.png</normaloff>../../Users/KOSA/Desktop/image/LightColor.png</iconset>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextUnderIcon</enum>
             </property>
             <property name="arrowType">
              <enum>Qt::NoArrow</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QSlider" name="SpecularSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Spot</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="OnPushButton">
             <property name="text">
              <string>On</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="OffPushButton">
             <property name="text">
              <string>Off</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="SpotColorButton">
             <property name="text">
              <string>Color</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../../Users/KOSA/Desktop/image/LightColor.png</normaloff>../../Users/KOSA/Desktop/image/LightColor.png</iconset>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextUnderIcon</enum>
             </property>
             <property name="arrowType">
              <enum>Qt::NoArrow</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="Line" name="line_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Light Position</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QRadioButton" name="XradioButton">
             <property name="styleSheet">
              <string notr="true">QRadioButton
{
	background-color:#fff;
	width: 100px;
    height: 30px;
}

QRadioButton::indicator 
{
    background-color: ff3399;
    border-style: outset;
    width: 0px;
    height: 0px;
}

QRadioButton::checked {
   background:#006 ;
 color:#fff;
}

QRadioButton::hover {
  background:#006 ;
 color:#fff;
}</string>
             </property>
             <property name="text">
              <string>X</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="YradioButton">
             <property name="styleSheet">
              <string notr="true">QRadioButton
{
	background-color:#fff;
	width: 100px;
    height: 30px;
}

QRadioButton::indicator 
{
    background-color: ff3399;
    border-style: outset;
    width: 0px;
    height: 0px;
}

QRadioButton::checked {
   background:#006 ;
 color:#fff;
}

QRadioButton::hover {
  background:#006 ;
 color:#fff;
}</string>
             </property>
             <property name="text">
              <string>Y</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="ZradioButton">
             <property name="styleSheet">
              <string notr="true">QRadioButton
{
	background-color:#fff;
	width: 100px;
    height: 30px;
}

QRadioButton::indicator 
{
    background-color: ff3399;
    border-style: outset;
    width: 0px;
    height: 0px;
}

QRadioButton::checked {
   background:#006 ;
 color:#fff;
}

QRadioButton::hover {
  background:#006 ;
 color:#fff;
}</string>
             </property>
             <property name="text">
              <string>Z</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QSlider" name="IntensityLightSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
